
# HG changeset patch
# User andrew
# Date 1357338114 0
# Node ID a57f192585241b5e3f004067f77ebffcfc534df5
# Parent  0685d07e5b432570a695af560f9dc7164a59a931
Include defs.make in buildtree.make so ZERO_BUILD is recognised and JVM_VARIANT_ZERO set.

diff -r 0685d07e5b43 -r a57f19258524 openjdk/hotspot/make/linux/makefiles/buildtree.make
--- openjdk/hotspot/make/linux/makefiles/buildtree.make	Thu Jan 03 15:32:49 2013 +0000
+++ openjdk/hotspot/make/linux/makefiles/buildtree.make	Fri Jan 04 22:21:54 2013 +0000
@@ -192,6 +193,7 @@
 	$(BUILDTREE_COMMENT); \
 	echo; \
 	echo "Platform_file = $(PLATFORM_FILE)" | sed 's|$(GAMMADIR)|$$(GAMMADIR)|'; \
+	echo "JVM_VARIANTS = $(JVM_VARIANTS)"; \
 	sed -n '/=/s/^ */Platform_/p' < $(PLATFORM_FILE); \
 	echo; \
 	echo "GAMMADIR = $(GAMMADIR)"; \

